pkg/kinesis/config.go

```diff
@@ -0,0 +1,13 @@
+package kinesis
+
+import "fmt"
+
+type Config struct {
+	AWSAccessKeyID     string
+	AWSSecretAccessKey string
+	Region             string
+}
+
+func (c *Config) GetIngestrURI() string {
+	return fmt.Sprintf("kinesis://?aws_access_key_id=%s&aws_secret_access_key=%s&region_name=%s", c.AWSAccessKeyID, c.AWSSecretAccessKey, c.Region)
```

Comment:
use `url` package to build the URI. 

This is unsafe, because access key id/secret access key may contain URI delimiters (for instance `=`,`&` etc).